cpp/tests/io/csv_test.cpp

```diff
@@ -2260,4 +2260,64 @@ TEST_F(CsvReaderTest, CsvDefaultOptionsWriteReadMatch)
   EXPECT_EQ(new_table_and_metadata.metadata.column_names[1], "1");
 }
 
+TEST_F(CsvReaderTest, UseColsValidation)
+{
+  std::string buffer = "1,2,3";
+
+  cudf::io::csv_reader_options idx_cnt_options =
```

Comment:
```suggestion
  const cudf::io::csv_reader_options idx_cnt_options =
```
use const at other places too.